modules/cloud-library/connect/cloud-library.php

```diff
@@ -8,7 +8,11 @@
 }
 
 class Cloud_Library extends Library {
-	const API_URL = 'https://my.elementor.com/api/v1';
+	const API_URL = 'https://my.elementor.com/cloud-library/api/v1';
```

Comment:
```suggestion
	const API_URL = 'https://my.elementor.com/api/v1/cloud-library';
```
I think this will be the endpoint (from the API Gateway)